src/main/java/com/parasoft/findings/jenkins/coverage/api/metrics/steps/CustomTableConfiguration.java

```diff
@@ -0,0 +1,67 @@
+package com.parasoft.findings.jenkins.coverage.api.metrics.steps;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.jenkins.plugins.datatables.TableConfiguration;
+import io.jenkins.plugins.util.JenkinsFacade;
+import jenkins.model.Jenkins;
+import org.apache.commons.lang3.StringUtils;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class CustomTableConfiguration extends TableConfiguration {
+
+    private final Map<String, Object> customConfiguration = new HashMap<>();
+
+    public void loadConfiguration() {
+        TypeReference<HashMap<String, Object>> typeRef = new TypeReference<>() {
+        };
+        String tableConfiguration = super.getConfiguration();
+        HashMap<String, Object> stringObjectHashMap;
```

Comment:
We can simplify the code here and rename the variable to have a more meaningful name.